openapi.yaml

```diff
@@ -5666,16 +5687,49 @@ components:
       type: string
       format: date-time
       description: >-
-        Date and time of creation, formatted as a [RFC
-        3339](https://www.rfc-editor.org/rfc/rfc3339.html) date-time.
+        Date and time of creation (for batch jobs: the status 'created' was set),
+        formatted as an [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339.html) date-time.
       example: '2017-01-01T09:32:12Z'
     updated:
       type: string
       format: date-time
       description: >-
-        Date and time of the last status change, formatted as a [RFC
-        3339](https://www.rfc-editor.org/rfc/rfc3339.html) date-time.
+        Date and time of the last status change,
+        formatted as an [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339.html) date-time.
+        If the status is `error`, `canceled` or `finished`,
+        this is the time when the job has ended.
+      example: '2017-01-01T09:54:18Z'
+    queued:
+      type: string
+      format: date-time
+      description: >-
+        Date and time of queueing the batch job (i.e., when the status 'queued' was set),
+        formatted as an [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339.html) date-time.
+      example: '2017-01-01T09:34:00Z'
+    started:
+      type: string
+      format: date-time
+      description: >-
+        Date and time when the batch job started processing (i.e., when the status 'running' was set),
+        formatted as an [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339.html) date-time.
       example: '2017-01-01T09:36:18Z'
+    expires:
+      type: string
+      format: date-time
+      description: >-
+        Time in UTC until which the assets and this document are accessible via the signed URL
+        that is provided with the relation type `canonical` in the links.
+        Formatted as an [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339.html) date-time.
+        After expiration, a document with new signed URLs can be retrieved through an authenticated
+        request to this endpoint.
+      example: '2017-02-01T09:54:18Z'
+    unpublished:
```

Comment:
I'm a bit hesitant about "unpublished". That word could suggest that the results still should be published, or can be recovered/republished. I think this should sound more irrevocable, like "removed", "erased" or "purged"

Reply:
The API doesn't describe whether that's permanent or not, it just tells the user that the data is currently not available.
Both cases are valid. In your case it's permanent, but it might not be in other cases.